cores/esp32/esp32-hal-log.h

```diff
@@ -37,6 +36,9 @@ extern "C"
 #define ARDUHAL_LOG_LEVEL CONFIG_ARDUHAL_LOG_DEFAULT_LEVEL
 #else
 #define ARDUHAL_LOG_LEVEL CORE_DEBUG_LEVEL
+#ifdef USE_ESP32_LOG
```

Comment:
I think that renaming the name to `USE_ESP_IDF_LOG` would describe a bit better what the switch does :)

Reply:
You are right, let me change it